tests/foreman/ui/test_discoveredhost.py

```diff
@@ -29,7 +29,7 @@
 
 
 @pytest.fixture(scope='module')
-def module_org():
+def discovery_org():
     org = entities.Organization().create()
```

Comment:
```suggestion
    org = default_sat.api.Organization().create()
```

Reply:
I changed some of these as I went, where there were only a few references in the module and I could change all of them and remove the import. If its alright with you, can I hold these additional changes to make in a bulk update against `master` branch?

Reply:
Yes, changing this in master for all and removing entities import, sounds good. 👍